docs/content/1.guide/0.getting-started.md

```diff
@@ -4,10 +4,14 @@ icon: ri:rocket-2-line
 
 # Getting Started
 
-Let's create a new Nitro app in few steps.
+Under the hood, Nitro is using [h3](https://h3.unjs.io) as HTTP framework, embeds many utilities (file routing, storage, cache...) and preconfigured packages like [unimports](https://unimports.unjs.io) or [unstorage](https://unstorage.unjs.io) and provide a set of presets to deploy your server anywhere.
```

Comment:
This is probably way too much for the first sentence 😅 (we might add it to last probably?)